wp-content/plugins/ig-hide-preview-button/plugin.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Plugin Name: Hide Preview Button
+ * Description: Hides the preview button when creating or editing posts
+ * Version: 1.0
+ * Author: Maximilian Ammann
+ * Author URI: https://github.com/Integreat
+ * License: MIT
+ * Text Domain: ig-hide-preview-button
+ */
+
+
+global $pagenow;
+
+if ('post.php' == $pagenow || 'post-new.php' == $pagenow) {
+    function wpse_125800_custom_publish_box() {
```

Comment:
an ig_ prefix for our own functions is also nice to have. Does not matter for this PR either, we're not going to keep this plugin forever (hopefully).